app/presenters/two_factor_authentication/selection_presenter.rb

```diff
@@ -36,13 +36,17 @@ def info
     def mfa_configuration_count; end
 
     def mfa_configuration_description
-      return '' if !disabled?
+      return '' if mfa_configuration_count.zero?
       t(
         'two_factor_authentication.two_factor_choice_options.configurations_added',
         count: mfa_configuration_count,
       )
     end
 
+    def mfa_added_label
+      "(#{mfa_configuration_description})"
+    end
+
     def disabled?
       false
     end
```

Comment:
I wonder if it might be simpler to let the base class handle most of the logic, if we can assume that the distinction is made on whether certain MFA methods only allow a single configuration. That way, we don't have to override so many methods in the subclasses.

Something like...

```rb
    def mfa_configuration_description
      return '' if mfa_configuration_count.zero?

      if single_configuration_only?
        t('two_factor_authentication.two_factor_choice_options.no_count_configuration_added')
      else
        t(
          'two_factor_authentication.two_factor_choice_options.configurations_added',
          count: mfa_configuration_count,
        )
      end
    end

    def mfa_added_label
      if single_configuration_only?
        ''
      else
        "(#{mfa_configuration_description})"
      end
    end

    def single_configuration_only?
      false
    end

    def disabled?
      single_configuration_only? && mfa_configuration_count > 0
    end
```

Then, `PivCacSelectionPresenter` looks more like:

```rb
module TwoFactorAuthentication
  class PivCacSelectionPresenter < SelectionPresenter
    def method
      :piv_cac
    end

    def single_configuration_only?
      true
    end

    def mfa_configuration_count
      user.piv_cac_configurations.count
    end
  end
end
```

What do you think?